environment/src/main/java/com/sequenceiq/environment/network/service/extended/MlxSubnetConstants.java

```diff
@@ -0,0 +1,14 @@
+package com.sequenceiq.environment.network.service.extended;
+
+public class MlxSubnetConstants {
+
+    public static final String MLX_SUBNET_MASK = "24";
+
+    public static final int MLX_SUBNET_STARTING_IP = 3;
```

Comment:
we dont need this if we are using count

Reply:
We don't need this because we are storing the subnet types

Reply:
done